package.json

```diff
@@ -6,7 +6,7 @@
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
     "@giscus/react": "^2.0.3",
-    "@mui/material": "^5.8.7",
+    "@mui/material": "^5.9.1",
```

Comment:
Is there any reason to upgrade the version?

Reply:
It's good to keep our packages up to date. However, let's have a separate issue for this. So that we can upgrade and run sanity along with it.